zpay32/invoice.go

```diff
@@ -171,6 +171,16 @@ type Invoice struct {
 	Features *lnwire.FeatureVector
 }
 
+// ToInvoiceExpiry returns the passed expiry if it is non-zero, otherwise
+// it returns the default one hour.
+func ToInvoiceExpiry(expiry time.Duration) time.Duration {
+	if expiry == 0 {
```

Comment:
What's this add? It also hard codes a _new_ default expiry rather than relying on the existing behavior to use the default expiry when decoding an invoice if unspecified.

Reply:
Removed.